ci/gpu/build.sh

```diff
@@ -73,9 +76,19 @@ conda list
 # BUILD - Build libcuml, cuML, and prims from source
 ################################################################################
 
+logger "Setting LD_LIBRARY_PATH to ${CONDA_PREFIX}/lib..."
```

Comment:
nit: to be more precise, you could just say `Adding "${CONDA_PREFIX}/lib" to LD_LIBRARY_PATH`

Reply:
yeah, definitely much better wording